assets/css/app.css

```diff
@@ -1,3 +1,3 @@
 
```

Comment:
I just added this css at the end 
`#bancontact_test_mode_description,#bancontact_live_mode_description_disabled{display:none;}`